CRM/Core/BAO/CustomGroup.php

```diff
@@ -945,135 +945,6 @@ public static function &getCustomGroupDetail($groupId = NULL, $extends = NULL, $
     return $groupTree;
   }
 
-  /**
-   * @deprecated since 5.71, will be removed around 5.85
-   */
-  public static function &getActiveGroups($entityType, $path, $cidToken = '%%cid%%') {
```

Comment:
this is called by uk.co.vedaconsulting.civisubscription but that seems abandoned